build_che.sh

```diff
@@ -14,11 +14,4 @@ if [ $? -ne 0 ]; then
   echo "Error building che/rh-che with dashboard"
   exit 1;
 fi
-cp target/export/che-dependencies/che/assembly/assembly-main/target/eclipse-che-*.tar.gz ${HomeDir}
```

Comment:
Why do you remove these lines ? don't you want to push the without dashboard image ? It's the one deployed to openshift afair ? 
IMO we should still build *without dashboard*. This also tests that the build is successful in both use-cases ( with and without dashboard).
If you don't want to push to docker the result of one of the 3 built distributions, you could change it in the `cico-build.sh` I assume. 
For example just remove the distributions you don't want to push to docker in line [39](https://github.com/l0rd/build-run-che/blob/19fa929597c3f920b15ec7224a95764564984604/cico_build.sh#L39)
Or add a `continue` statement in the case you don't want to manage (at line [52](https://github.com/l0rd/build-run-che/blob/19fa929597c3f920b15ec7224a95764564984604/cico_build.sh#L52) if I understood correctly)